tests/e2e/concurrent_query_modules/test_query_modules/module_test.cpp

```diff
@@ -0,0 +1,59 @@
+// Copyright 2023 Memgraph Ltd.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt; by using this file, you agree to be bound by the terms of the Business Source
+// License, and you may not use this file except in compliance with the Business Source License.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+#include <mgp.hpp>
+
+#include <chrono>
+#include <cmath>
+#include <list>
+#include <thread>
+
+const char *kProcedureHackerNews = "hacker_news";
+const char *kArgumentHackerNewsVotes = "votes";
+const char *kArgumentHackerNewsItemHourAge = "item_hour_age";
+const char *kArgumentHackerNewsGravity = "gravity";
+const char *kReturnHackerNewsScore = "score";
+
+void HackerNews(mgp_list *args, mgp_graph *memgraph_graph, mgp_result *result, mgp_memory *memory) {
+  mgp::MemoryDispatcherGuard guard(memory);
```

Comment:
we will also need to update MAGE docs because the way we have the How To Guide in the docs, they are suggesting the old way of handling the memory pointeer.